test/scripts/products/h24-verify-selfmanaged-apicast-and-custom-policy/test.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+#  Prerequisites:
+# 		oc logged as cluster admin (kubeadmin)
+#  Recommended, before running current version of script, to have following:
```

Comment:
I think the part of this PR should be also an update to the [manual test case](https://github.com/integr8ly/integreatly-operator/blob/master/test-cases/tests/products/h24-verify-selfmanaged-apicast-and-custom-policy.md) with instructions on how to run this script (similarly like it was done [for the h22 test](https://github.com/integr8ly/integreatly-operator/blob/master/test-cases/tests/products/h22-validate-that-rate-limit-service-is-working-as-expected.md))

Reply:
Done. Automation test instructions added to doc.